bin/reth/src/cli.rs

```diff
@@ -171,6 +175,28 @@ impl Verbosity {
     }
 }
 
+/**
+ * Display hardfork information on startup
+ * 
+ * Example:
+ * Hardforks:
+ *   - Homestead: Block 0
+ *   - London: Block 1
+ *   - Paris: TTD 1000
+ *   - Shanghai: Timestamp 203023214012
+ * 
+ *  */ 
+
+fn display_hardforks_on_startup() {
+
+    let hardfork = MAINNET.hardforks(); 
+
+    println!("Hardforks:");
+    for (k,v) in &*hardfork{
+        println!("      {}      : {}", k, v);
+    }
```

Comment:
ptal at how to format/align 

https://doc.rust-lang.org/std/fmt/#formatting-parameters